service/transform-service.py

```diff
@@ -89,26 +92,41 @@ def generate(entities, endpoint):
             for index, entity in enumerate(entities):
                 if index > 0:
                     yield ","
-                url_per_entity, method_per_entity, headers_per_entity, prop_per_entity = url, method, headers, prop
+                url_per_entity, method_per_entity, headers_per_entity, prop_per_entity, tolerable_status_codes_per_entity, payload_property_per_entity = url, method, headers, prop, tolerable_status_codes, payload_property
```

Comment:
2 new config items are added

Reply:
Can't find the declaration of `payload_property`, so I'm guessing `payload_property_per_entity` will be "None" until it's set further down at line 103.

As far as I can tell this won't cause any harm, just wanted to check with you if the naming is intentional as the only other reference to "payload_property" is the removed line 102.

Reply:
line 21 is where `payload_property` is declared. One of us is looking at the wrong place ,i suppose?

Reply:
My bad